packages/strapi-plugin-graphql/services/Mutation.js

```diff
@@ -183,6 +183,18 @@ module.exports = {
         }),
       });
 
+      if (options.input && options.input.where) {
```

Comment:
Can you please keep context and ctx separated. 

You can set params and body on que context like previsouly but before cloning it to create the policy context. 

This will avoid having the ctx.request.graphql after the policies are run

Reply:
Ok will do so tomorrow!